pages/api/og.tsx

```diff
@@ -43,22 +9,21 @@ export const config = {
 export default function (req: NextRequest) {
   try {
     const { searchParams } = new URL(req.url);
-    const testUrl = new URL(req.url);
     const hasTitle = searchParams.has('title');
     const title = hasTitle
       ? searchParams.get('title')?.slice(0, 100)
       : 'My Store';
     const hasColor = searchParams.get('color')!;
     const color = hasColor ? hasColor : '000000';
-    const foreground =
-      lightOrDark(`#${color}`) == 'light' ? '000000' : 'FFFFFF';
-    const logo = searchParams.get('logo');
+    const hasForeground = searchParams.get('backgroundColor')!;
+    const foreground = hasForeground;
```

Comment:
As `backgroundColor` is a new field, should you maintain the default (lightOrDark()) approach if it's not found?  As these are different repos, the changes can't be released atomically, so we should maintain backwards compatibility until it's not needed.